### PR TITLE
Update commons-ui

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**

### CODE REVIEW

File: src/components/top-bar-equipment-seach-dialog/custom-suffix-renderer.tsx [hunk 1]
```diff
@@ -76,19 +74,18 @@ export const CustomSuffixRenderer: FunctionComponent<
...
 
-    const openNetworkAreaDiagramCB = useCallback(
-        (
-            e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-            element: EquipmentInfos
-        ) => {
+    const openNetworkAreaDiagramCB = useCallback<
+        MouseEventHandler<HTMLButtonElement>
+    >(
+        (e) => {
```
Comment (thangqp) [inline src/components/top-bar-equipment-seach-dialog/custom-suffix-renderer.tsx:83]:
```suggestion
    const openNetworkAreaDiagramCB = useCallback(
        (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```
I have no type checking error with the code above

File: src/components/top-bar-equipment-seach-dialog/custom-suffix-renderer.tsx [hunk 2]
```diff
@@ -48,11 +47,10 @@ export const CustomSuffixRenderer: FunctionComponent<
...
 
-    const centerOnSubstationCB = useCallback(
-        (
-            e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-            element: EquipmentInfos
-        ) => {
+    const centerOnSubstationCB = useCallback<
+        MouseEventHandler<HTMLButtonElement>
+    >(
+        (e) => {
```
Comment (thangqp) [inline src/components/top-bar-equipment-seach-dialog/custom-suffix-renderer.tsx:53]:
```suggestion
    const centerOnSubstationCB = useCallback(
        (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```
I have no type checking error with the code above

File: src/components/top-bar-equipment-seach-dialog/custom-suffix-renderer.tsx (hunk 1 above)
Comment (Tristan-WorkGH) [reply src/components/top-bar-equipment-seach-dialog/custom-suffix-renderer.tsx:83]: the error happen on the "IconButton.onClick":
> TS2345: Argument of type MouseEvent<HTMLButtonElement, MouseEvent> is not assignable to parameter of type
>> MouseEvent<HTMLButtonElement, MouseEvent<Element, MouseEvent>>

**Review by thangqp — APPROVED** [review]
Test OK, code OK

**Pull request merged**